test/dummy/app/views/dummy/ui/embed.slim

```diff
@@ -0,0 +1,18 @@
+.container-fluid
+  h1
+    a.d-unlink href=dummy_ui_path UI kit
+
+  - %w[light dark].each do |theme|
+    .bg-body.p-g.mt-g.border.rounded data-bs-theme=theme
+      h2.mt-0 Embed - #{theme}
+
+      - embed_list = [ \
```

Comment:
@mreq tohle asi není úplně ideální, minimálně z hlediska délky řádku, ale nepovedlo se mi to zalomit